internal/telemetry/client.go

```diff
@@ -47,10 +49,16 @@ var (
 		},
 		Timeout: 5 * time.Second,
 	}
-	// TODO: Default telemetry URL?
 	hostname string
 )
 
+// Default values for determining where to upload telemetry
+const (
+	DefaultAgentAddress            = "http://localhost:8126"
+	AgentTelemetryEndpoint         = "/telemetry/proxy/api/v2/apmtelemetry"
+	DefaultDirectTelemetryEndpoint = "https://instrumentation-telemetry-intake.datadoghq.com/"
```

Comment:
I don't think we need these exported. We can always do it later if we do, right?

Reply:
Sure, that makes sense to me.